### DISCUSSION TITLE
Can someone help me integrated this in a CRA application and to make a standalone node application of python language server

### DISCUSSION THREAD

**Discussion by ArishAzmat** [TypeFox/monaco-languageclient#792] [Q&A] [answered]
Implementing this in a standalone react application (cra) and using a standalone python without other things is difficult for me.
There so many packages being used here and I am getting bunch of errors

Can anyone guide me how and what files and function should I extract from this repo to create a python server and react app

**Comment by kaisalmen** [comment]
HI @ArishAzmat do you need to use create react app? Why not start with vite's templates, see here: https://github.com/vitejs/vite/tree/main/packages/create-vite

**Reply by ArishAzmat** [reply]
Unfortunately, I can't since I already am working on a project (which is based on CRA) and I need to introduce python [& R] language server in it as an enhancement.
btw I was able to make some functionality work but facing one issue with ```useWorkerFactory```. Is there any non-hook function which I can use as in replacement for ```useWorkerFactory```. It is causing me this error ```React Hook "useWorkerFactory" is called in function "configureMonacoWorkers"``` (or is there a way to avoid this).

**Reply by kaisalmen** [reply, marked as answer]
Hi @ArishAzmat just comment `monacoWorkerFactory` in the configuration for now. The editor will run on main (warning is printed in browser console). Does it startup-up then?

**Reply by ArishAzmat** [reply]
Hi @kaisalmen 
Yes, it is running with warning.
Is it not an important thing?

**Reply by kaisalmen** [reply]
It makes the editor code run in a web worker. What I don't understand it why it can't be loaded in your application. This is just works with `typefox/monaco-editor-react` in our examples here.

**Reply by ArishAzmat** [reply]
Idk man maybe it's because of CRA setup.

**Reply by kaisalmen** [reply]
Yes, that maybe possible or there is a problem with the webpack configuration.
